src/include/common/string_utils.h

```diff
@@ -36,6 +36,7 @@ class KUZU_API StringUtils {
     static bool CharacterIsSpace(char c) {
         return c == ' ' || c == '\t' || c == '\n' || c == '\v' || c == '\f' || c == '\r';
     }
+    static bool CharacterIsNewLine(char c) { return c == '\n' || c == '\r'; }
```

Comment:
our naming is convension is to use camel case, can you rename to `characterIsNewLine`?

Reply:
updated